advanced/events.md

```diff
@@ -321,32 +321,34 @@ tinymce.init({
 
 ### ExecCommand
 
-This even is fired when a command like Bold/Italic etc is made by the editor.
+This event is fired when a command like Bold/Italic etc is made by the editor.
 
 #### Parameters
-* **content** `String` - The HTML content that's being extracted from the editor.
+* **content** `String` - The name of the command that was executed.
```

Comment:
change **content** `String`
to **command** `String`

Reply:
Done